docs/astro/files/operators/get_file_list.rst

```diff
@@ -0,0 +1,14 @@
+=============
+get_file_list
+=============
+
+.. _get_file_list:
```

Comment:
@pankajastro We need to move this to the top of the page before H1, otherwise when we use this **:ref:`get_file_list`** in doc the sphinx doc will print "When to use the ``get_file_list`` operator" and not "get_file_list".

Reply:
actually, I was looking at other examples that we have added https://github.com/astronomer/astro-sdk/blame/main/docs/astro/sql/operators/append.rst#L7 .

do you want to move  `.. _get_file_list` on top of file?

how does it resolve when we have multiple objects in the same rst file?

Reply:
as discussed fixed it.